vtex-apps/login
# Fix/messages

#### What is the purpose of this pull request?
Normalize messages that were inconsistent through languages

#### What problem is this solving?
This fixes all the app messages in english and translate them to `pt` and `es`, passing through designer Davi Shamá and Vladimir's revision.

#### How should this be manually tested?
You can see the login app running with the new messages at
https://rafaprtest--storecomponents.myvtex.com/

![image](https://user-images.githubusercontent.com/22064061/65998889-a3fa3800-e472-11e9-9bac-40d1d0162c9e.png)

#### Types of changes
